SSB - Teknologiradar.csv

```diff
@@ -8,3 +8,7 @@ JupyterLab,Ta i bruk,Klargjøring,FALSE,"Notebook basert kodemiljø for å skriv
 Google Dataflow,Vurder,Klargjøring,FALSE,"DataFlow bygger på Apache Beam teknologi som egner seg godt for å lage automatiserte batch og streaming pipelines (klargjøringsjobber). DataFlow er skalerer godt, tilbyr egen ""forvaltningsfri"" infrastruktur med for distribuert prosessering. Kan også integreres med Notebook teknologi for å skrive produksjonskode.<br>Mer info her:<a href=""https://cloud.google.com/dataflow"">DataFlow</a>"
 Grafana,Ta i bruk,Utvikling - infrastruktur,FALSE,"Teknologi for å visulalisere data fra mange ulike kilder. Mest brukt til å visualisere logger og metrikker. Brukerne lage egne fleksible dashboard. <br>Mer info her:<a href=""https://grafana.com"">Grafana</a>"
 Delete4Fun,Avvent,Utvikling - infrastruktur,TRUE,"Teknologi for å slette infrastruktur for moroskyld!"
+PowerBI,Prøv ut,Analyse - formidling,TRUE,"Teknologi for visualisering"
+PowerBI,Prøv ut,Klargjøring,TRUE,"Teknologi for visualisering"
+MOVEit,Ta i bruk,Datainnsamling - lagring,FALSE,"Overføring av filer på bakkenivå"
+Git,Ta i bruk,Utvikling - infrastruktur,FALSE,"Teknologi for å koordinere prosjekter"
```

Comment:
Tenker du her på Git eller Github? Sjekk denne siden: https://blog.devmountain.com/git-vs-github-whats-the-difference/

Reply:
Tenkte på Github, sender ny PR

Reply:
Flott! Jeg fjernet PowerBI da den allerede er lagt inn via en annen PR (pull request)